packages/plugin-ext/src/main/browser/status-bar-message-registry-main.ts

```diff
@@ -46,31 +44,23 @@ export class StatusBarMessageRegistryMainImpl implements StatusBarMessageRegistr
         };
 
         this.entries.set(id, entry);
-        return this.delegate.setElement(id, entry).then(() => Promise.resolve(id));
+        await this.delegate.setElement(id, entry);
     }
 
-    $update(id: string, message: string): void {
+    $update(id: string, text: string): void {
```

Comment:
could you please avoid to rename the fields everywhere, very hard to review 
it seems you only add an await